tests/end-to-end/multivariate/test_base.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import pytest
+
+from copulas.datasets import sample_trivariate_xyz
+from copulas.multivariate import GaussianMultivariate, VineCopula
+
+
+class TestMultivariate():
+
+    @pytest.mark.parametrize('model', [GaussianMultivariate, VineCopula])
+    def test_fixed_random_state(self, model):
+        """Test that the multivariate models work with a fixed seed.
+
+        Expect that fixing the seed generates a reproducable sequence
+        of samples. Expect that these samples are different from randomly
+        sampled results.
+        """
+        data = sample_trivariate_xyz()
+        if model == VineCopula:
```

Comment:
We can skip this `if/else` block by parameterizing with instances instead of classes:

```python
@pytest.mark.parametrize('model', [GaussianMultivariate(), VineCopula('direct')])
```